---
uav_simulator/Obstacle data
bool success
